helpers/constants.py

```diff
@@ -141,7 +141,7 @@
 xref_validation_module_name = "at-uploader | This doc contains a list of refs"
 # xref validation constants in CP
 Xref_linkText1 = "at-uploader | Assembly Publish test"
-Xref_linkText2 = "Content Test Module | Logging in to the web console using Kerberos authentication (Image present)"
+Xref_linkText2 = "at-uploader | Module bulk operations 4"
```

Comment:
Please update this as per changes suggested here https://gitlab.cee.redhat.com/ccs-tools/uploader-sample-repo-pantheon2/-/merge_requests/11/diffs

Reply:
Changed uploader repo's hyperlink "Different module included in the same assembly" to neglect  "bulk operations" substring.